templates/lune/partials/product/product.snip.html

```diff
@@ -0,0 +1,175 @@
+{{#product}}
+  <main role="main" class="lune-page lune-page-product {{classes}}">
+    <div class="lune-page-product-wrapper md-col-9 mx-auto">
+
+      {{#hero}}
+        <section class="lune-product-section lune-product-section-hero relative z1 flex">
+          <div class="lune-hero-content col-12 md-mt0 flex flex-column items-center justify-center">
+            <div class="lune-hero-title-wrapper">
+              <h1 class="lune-hero-title relative caps m0 px3 md-p0 z1 lune-anim-move-up {{#title_parts.1}}lune-title-has-mobile-hyphen{{/title_parts.1}}" aria-label="{{full_title}}">{{title_parts.0}}</h1>
+              {{#title_parts.1}}
+                <p class="lune-hero-title caps m0 relative px3 md-p0 z4 lune-anim-move-up" aria-hidden>{{.}}</p>
+              {{/title_parts.1}}
+            </div>
+
+            {{#image}}
+              <figure class="lune-hero-image-wrapper z3 flex justify-center col-12 m0">
+                <!-- WebP image -->
+                <amp-img
+                  class="lune-anim-scale-up lune-hero-image z2 col-7 md-mt0"
+                  alt="{{alt}}"
+                  srcset="{{webp_srcset}}"
+                  width="{{width}}"
+                  height="{{height}}"
+                  layout="responsive">
+                    <!-- PNG Fallback when WebP is not supported -->
+                    <amp-img
+                      fallback
+                      class="lune-anim-scale-up"
+                      alt="{{alt}}"
+                      srcset="{{fallback_srcset}}"
+                      width="{{width}}"
+                      height="{{height}}"
+                      layout="responsive">
+                    </amp-img>
+                </amp-img>
+              </figure>
+            {{/image}}
+          </div>
+
+          <!-- Scroll indicator -->
```

Comment:
I don't think a `footer` is suitable for the scroll indicator. From MDN: 

> A footer typically contains information about the author of the section, copyright data or links to related documents.

I'll change to an `aside` which would be more suitable